src/profile-logic/profile-data.js

```diff
@@ -1507,6 +1507,47 @@ export function isNetworkMarker(marker: TracingMarker): boolean {
   return !!(marker.data && marker.data.type === 'Network');
 }
 
+export function getNetworkMarkers(thread: Thread): Array<NetworkPayload> {
+  const { markers } = thread;
+  const networkMarker = [];
+
+  // making sure we return all markers after we got all from type network
+  if (!markers) {
+    return networkMarker;
+  }
+
+  // filter for all markers
+  for (let markerIndex = 0; markerIndex < markers.length; markerIndex++) {
+    const data = markers.data[markerIndex];
+    if (!data || data.type !== 'Network') {
+      continue;
+    }
+
+    // Match start marker with stop / redirect, if they have one.
+    if (
+      (!!networkMarker && data.status === 'STATUS_STOP') ||
+      (!!networkMarker && data.status === 'STATUS_REDIRECT')
+    ) {
+      const matchURI =
+        markers.data[markerIndex].URI === markers.data[markerIndex - 1].URI;
```

Comment:
I think this line will throw an error if markerIndex is equal to 0.

Reply:
Fox context: Usually the `STATUS_STOP` marker always has a `STATUS_START` marker, so It should never hit `markerIndex === 0`. 

But you are right, the moment we change this it will very likely break.